docs/user/README.md

```diff
@@ -159,6 +159,45 @@ nodes:
   replicas: 2
 ```
 
+kind also supports multiple control-plane and HA setups using a stacked etcd topology:
+```yaml
+# six node cluster with three control-plane and three worker nodes
+kind: Config
+apiVersion: kind.sigs.k8s.io/v1alpha2
+nodes:
+- role: control-plane
+  replicas: 3
+- role: worker
+  replicas: 3
+- role: external-load-balancer
+```
+
+The `external-load-balancer` node is [HAProxy][haproxy] based and it acts like a load
```

Comment:
nit: "...node is HAProxy based" sounds a bit awkward. Does this mean that the node image is based on a HAProxy image?

Reply:
Currently the fact that it is haproxy based is pretty internal, we don't make any guarantees about this yet. It could very well be something else in the future

Reply:
I recall at some point hearing from the sig-cluster-lifecycle people that kind having a HAproxy LB was a big deal. This detail may be important but Lubomir would know more about it.

Reply:
i don't think it's a big deal. it's just a decent LB solution.